pepper-apis/gae-egress-proxy/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM ubuntu:focal-20200729
```

Comment:
Ubuntu images are from here: https://hub.docker.com/_/ubuntu. Using focal 20.04 since that has the latest squid v4.10.

Reply:
Looks good!